.github/workflows/dist.yml

```diff
@@ -1,31 +1,57 @@
 name: dist-check
 
-on: [push, pull_request]
+on:
+  # Manually triggerable in github
+  workflow_dispatch:
+
+  # When a push occurs on either of these branches
+  push:
+    branches:
+      - master
+      - development
+
+  # When a push occurs on a PR that targets these branches
+  pull_request:
+    branches:
+      - master
+      - development
+
+  schedule:
```

Comment:
actually why would you like this workflow on a schedule?

Reply:
There's no real need, there's also no real need to have it on anything but push to master or development. This workflow actually does very little compared to `pytest.yaml`, which is also build a dist version and then runs tests on it. The only difference here is that it runs [`twine check`](https://twine.readthedocs.io/en/latest/#twine-check) which checks that the long description will parse correctly on PyPi, that's it.

I would probably remove this workflow entirely upon second though.

Reply:
Also, I think the mypy compliance thing is wrong, I originally submitted a PR to autosklearn to check for it which I think got lost in translation to the workflow files in all our repos. It should be `mypy -c "import autosklearn"`

Reply:
Fixed the compliance check